server/controllers/list.js

```diff
@@ -1794,11 +1794,22 @@ const getAvailableLists = (req, resp) => {
     isArchived: false
   };
 
-  List.find(query, 'name')
-    .lean()
-    .exec()
-    .then(docs => (docs ? resp.send(docs) : resp.sendStatus(400)))
-    .catch(() => resp.sendStatus(400));
+  try {
+    const lists = await List.find(query, 'name')
+      .lean()
+      .populate('cohortId', 'isArchived')
+      .exec();
+
+    const listToSend = lists
+      .filter(
+        list => !list.cohortId || (list.cohortId && !list.cohortId.isArchived)
+      )
+      .map(({ _id, name }) => ({ _id, name }));
+
+    resp.send(listToSend);
+  } catch {
```

Comment:
Please at least log the error to ease the debugging.

Reply:
We don't console log errors. But I added a task: https://jira2.sanddev.com/browse/EOC-525,
to discuss this and create some logging system or at least console.log errors.